hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenAccountWipeHandler.java

```diff
@@ -206,7 +206,7 @@ private ValidationResult validateSemantics(
 
         final var accountRel = TokenHandlerHelper.getIfUsable(accountId, tokenId, tokenRelStore);
         validateFalse(
-                token.treasuryAccountNumber() == accountRel.accountNumber(),
+                token.treasuryAccountId().accountNum() == accountRel.accountId().accountNum(),
```

Comment:
Need to compare IDs instead
```suggestion
                token.treasuryAccountId().equals(accountRel.accountId())
```

Reply:
Updated, thanks.